walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,33 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
+    Collection<TimeRange> availibility;
```

Comment:
This makes `availability` a "member" variable for your class.  Remember with variable scoping, the variable will be accessible to all places within the curly braces it was defined in.  This means `availability` will be accessible throughout your FindMeetingQuery.  More importantly, it'll be accessible outside the `query()` method.

If there's no specific reason for this variable to be accessible outside of `query()`, I would suggest moving into `query(){...}` instead to limit the scope